package.json

```diff
@@ -10,13 +10,14 @@
   "module": "dist/index.modern.js",
   "source": "src/index.ts",
   "engines": {
-    "node": ">=10"
+    "node": ">=14.0.0"
   },
   "scripts": {
-    "build": "microbundle-crl --no-compress --format modern,cjs",
```

Comment:
don't get why we didn't compress stuff before, managed to  decrease bundle size from 100KB to 70